builds.yml

```diff
@@ -59,6 +59,14 @@ buildTypes:
     isPrerelease: true
     manifestOverrides: ./app/build-types/flask/manifest/
 
+  blockaid:
```

Comment:
I think what you want is just a feature called `blockaid`, not a buildType

Reply:
We do not want this feature to be part of flask, only adding feature and not including it in buildType is breaking build

Reply:
We have not introduced new build type in the PR, only a new feature